src/store/blockchain/reducer.ts

```diff
@@ -1,9 +1,10 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Environment } from 'src/utils';
 import { Blockchain } from './type';
 
 const initialState: Blockchain = {
     latestBlock: 0,
-    chainId: 0,
+    chainId: process.env.SF_ENV === Environment.PRODUCTION ? 42161 : 421614,
```

Comment:
is there a reason not to use isProdEnv here?
Please use arbitrumSepolia.id and arbitrum.id